src/tools/pub/cmd/pub-get.md

```diff
@@ -145,6 +145,13 @@ By default, pub precompiles executables
 in immediate dependencies (`--precompile`).
 To prevent precompilation, use `--no-precompile`.
 
+### `--enforce-lockfile`
+
+Fetches dependencies, but will refuse to resolve
+if anything deviates from `pubspec.lock` or there is no `pubspec.lock`.
+This is useful for deploying,
+and ensuring reproducible runs in CI and production. 
```

Comment:
**issue:** Tighten this language to ease readability and match later revised copy.

See your copy in [this file](https://github.com/dart-lang/site-www/pull/4407/files#diff-f24f8dca36386ceb9bc6e11d49921934f1486e12084bed60f621cd772348cd41R464-R471).

**question:** What does "resolve" mean here and in the linked copy? What is being resolved?

Reply:
Added the part about is causing an error as opposed to just a warning. 

> question: What does "resolve" mean here and in the linked copy? What is being resolved?

I was going to add a link to an explanation of package resolution elsewhere in the docs, but I realized the link I would've added is _to this page_. `pub get` itself _is the definition_ for package resolution; everything it does is what package resolution...is. Linking to the page we're already on seemed like overkill. We can assume that someone reading about `--enforce-lockfile` knows what "resolve" means in this context.